src/components/ui/payment-history.tsx

```
@@ -0,0 +1,126 @@
+                                }
+                                if (a.payments.length > 0 && b.payments.length === 0) {
+                                    return 1
+                                }
+                                if (a.payments.length === 0 && b.payments.length === 0) {
+                                    return 0
+                                }
+
+                                const latestPaymentA = a.payments.sort((a, b) => {
+                                    return new Date(b.transaction_time).getTime() - new Date(a.transaction_time).getTime()
+                                }
+                                )[a.payments.length - 1] as any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)